StitchCore/StitchCore/Source/Network/Adapter/StitchNetworkAdapter.swift

```diff
@@ -29,15 +29,9 @@ public class StitchNetworkAdapter: NetworkAdapter {
         request.allHTTPHeaderFields = contentHeaders
         request.httpMethod = method.rawValue
 
-        if method != .get, let parameters = parameters {
-            guard let jsonData = try? JSONEncoder().encode(parameters) else {
-                printLog(.error, text: "bad json")
-                task.result = .failure(StitchError.illegalAction(message: "bad json"))
-                return task
-            }
-
-            printLog(.debug, text: String(data: jsonData, encoding: .utf8)!)
-            request.httpBody = jsonData
+        if method != .get, let data = data {
+            printLog(.debug, text: String(data: data, encoding: .utf8)!)
```

Comment:
do we want this log here still?

Reply:
Depends. It's helpful when running tests, and if the user actually uses our logging module, it could be useful for them as well. We're not really taking advantage of diagnostics at all.

Reply:
okay. I'm just thinking of security where we could be logging sensitive details.